assets/js/base/components/loading-mask/style.scss

```diff
@@ -1,6 +1,7 @@
 .wc-block-components-loading-mask {
 	position: relative;
 	min-height: 18px + $gap;
+	pointer-events: none;
```

Comment:
This prevents mouse interaction with a blocked area.

Reply:
> Elements with pointer-events: none will still receive focus through sequential keyboard navigation using the Tab key.

This blocks mouse interaction only I believe.

Now that loading mask is present, and reflects state if it is confirmed as an edge case issue, we can tackle this in the loading mask component as a follow-up.